### ISSUE TITLE
[BUG] Modual Makes to many requests pers second

### ISSUE THREAD

**Issue by Zarhog** [CDeenen/LockView#90]
The modual causes a Spam of requests which in turn causes the server to lag and ping to skyrocket

**Comment by CDeenen** [comment]
Yes the module sends a lot of requests (especially if you enable Viewboxes), because that results in a smooth experience for the GM. But I host my games on a Raspberry Pi 4, so not really powerful hardware, and this has never been an issue for me.
It is possible that another module is interfering and causing Lock View to send a lot more requests than it should. Could you try disabling all other modules to see if it persists?

I could also look into adding a low performance option, where it reduces the amount of requests it sends.
